comid/flagsmap.go

```diff
@@ -179,6 +180,24 @@ func (o *FlagsMap) Get(flag Flag) *bool {
 	}
 }
 
+func (o *FlagsMap) Compare(r *FlagsMap) error {
+	claim, err := o.MarshalJSON()
```

Comment:
JSON conversion seems to be  an unncessarily round-about way of doing this. Also, doing byte comparisons of JSON is always iffy. I think this is _probably_ fine in this case, however, if `Extensions` happen to contain maps for some reason, they may not always serilize the same way.